go-apps/meep-rnis/server/rnis.go

```diff
@@ -357,54 +401,85 @@ func reInit() {
 	_ = rc.ForEachJSONEntry(keyName, repopulateRrSubscriptionMap, nil)
 	_ = rc.ForEachJSONEntry(keyName, repopulateMrSubscriptionMap, nil)
 	_ = rc.ForEachJSONEntry(keyName, repopulateNrMrSubscriptionMap, nil)
-
 }
 
 // Run - Start RNIS
 func Run() (err error) {
+	// Start MEC Service registration ticker
+	if appEnablementEnabled {
+		startRegistrationTicker()
+	}
 	return sbi.Run()
 }
 
 // Stop - Stop RNIS
 func Stop() (err error) {
-	retryAppEnablementTicker.Stop()
+	// Stop MEC Service registration ticker
+	if appEnablementEnabled {
+		stopRegistrationTicker()
+	}
 	return sbi.Stop()
 }
 
-func getAppInstanceId(appName string, appVersion string) string {
-	//var client *appInfoClient.APIClient
-	appInfoClientCfg := appInfoClient.NewConfiguration()
-	appInfoClientCfg.BasePath = appEnablementClientUrl + "/app_info/v1"
+func startRegistrationTicker() {
+	// Wait a few seconds to allow App Enablement Service to start
+	log.Info("Waiting for App Enablement Service to start")
+	time.Sleep(5 * time.Second)
```

Comment:
Same comment as Location Service for the 5sec sleep

Reply:
Same response.